packages/components/src/mobile/bottom-sheet/cell.native.js

```diff
@@ -196,22 +204,28 @@ class BottomSheetCell extends Component {
 					accessibilityHint
 				}
 				onPress={ onCellPress }
-				style={ { ...styles.clipToBounds, ...style } }
+				style={ [ styles.clipToBounds, style ] }
 			>
 				{ drawTopSeparator && (
 					<View style={ separatorStyle() } />
 				) }
-				<View style={ styles.cellContainer } pointerEvents={ containerPointerEvents }>
-					<View style={ styles.cellRowContainer }>
-						{ icon && (
-							<View style={ styles.cellRowContainer }>
-								<Dashicon icon={ icon } size={ 24 } color={ iconStyle.color } />
-								<View style={ platformStyles.labelIconSeparator } />
-							</View>
-						) }
-						<Text numberOfLines={ 1 } style={ [ defaultLabelStyle, labelStyle ] }>
-							{ label }
-						</Text>
+				<View style={ cellContainerStyles } pointerEvents={ containerPointerEvents }>
+					<View style={ rowContainerStyles }>
+						<View style={ styles.cellRowContainer }>
+							{ icon && (
+								<View style={ styles.cellRowContainer }>
+									<Dashicon icon={ icon } size={ 24 } color={ iconStyle.color } />
+									<View style={ platformStyles.labelIconSeparator } />
+								</View>
+							) }
+							<Text numberOfLines={ 1 } style={ [ defaultLabelStyle, labelStyle ] }>
```

Comment:
@etoledom If we remove `numberOfLines={ 1 }` we can have a not truncated label and it will look like

<img src="https://user-images.githubusercontent.com/22746080/68469880-52845c00-021b-11ea-9d74-dc539101cfd0.png" width="300" />

Reply:
I've also checked how it will affect other settings:

before | after
--- | ---
![Screenshot 2019-11-08 at 11 36 33](https://user-images.githubusercontent.com/22746080/68470591-a3e11b00-021c-11ea-8605-b5e6666e4d7a.png) | ![Screenshot 2019-11-08 at 11 36 28](https://user-images.githubusercontent.com/22746080/68470598-a774a200-021c-11ea-8fff-47b7887173b7.png)

Reply:
Looks better than cropping! 
Let's go with it if it does't affect other instances of cells and controls.

Reply:
> I've also checked how it will affect other settings

Again, better than cropping. Looks good to me :)